src/pudl/output/ferc1.py

```diff
@@ -2274,36 +2274,45 @@ def annotated_forest(self: Self) -> nx.DiGraph:
         annotated_forest = deepcopy(self.forest)
         nx.set_node_attributes(annotated_forest, self.node_attrs)
         nx.set_edge_attributes(annotated_forest, self.edge_attrs)
-        annotated_forest = self.propagate_node_attributes(annotated_forest)
 
         logger.info("Checking whether any pruned nodes were also tagged.")
         self.check_lost_tags(lost_nodes=self.pruned)
         logger.info("Checking whether any orphaned nodes were also tagged.")
         self.check_lost_tags(lost_nodes=self.orphans)
-        self.check_conflicting_tags(annotated_forest)
+
+        annotated_forest = self.propagate_node_attributes(annotated_forest)
         return annotated_forest
 
     def propagate_node_attributes(self: Self, annotated_forest: nx.DiGraph):
         """Propagate tags.
 
-        Propagate tags leafwards, rootward &  to the _correction nodes.
+        Propagate tags root-ward, leaf-wards &  to the _correction nodes. We
+        propagate the tags root-ward first because we primarily manually
+        compiled tags for the leaf nodes, so we want to send those leafy tags
+        root-ward first before trying to send tags back up the graph.
```

Comment:
are root-ward and leaf-ward a play on "forward" and "backward"? It's a little confusing. Maybe say "starting from X" or "from X to Y". I'm not sure I understand the sentences with those phrases.

Reply:
It looks like these are commonly used terms in the module. Maybe it's not worth changing all of them? As long as there is a clear definition somewhere.

Reply:
A clarifying question - why are manually compiled tags prioritized?

Reply:
yes the tree, root, leaf terminology is more or less grafted directly from [networkx](https://networkx.org/documentation/stable/reference/algorithms/tree.html) which is the primary tool we are using to grab the most granular components of the financial tables to feed into the rate base table.

the tags are entirely manually compiled but we don't manually compile the tags for ever single `xbrl_factoid` across the detailed tables because that would be *alot*. We use this propagation step to spread the tags we have compiled.